OCP/OCP_formulation.m

```diff
@@ -595,6 +595,7 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 % Scale cost function
 Jall_sc = sum(Jall)/dist_trav_tot;
+opti.minimize(Jall_sc);
```

Comment:
not clear to me why this changed location. This is not related to changes in the settings?

Reply:
It was in the code section for passing options to the solver, so I moved it to the section that finalizes the cost function.